backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -93,6 +93,9 @@ def resolve_fetch_failures_0_members(
         # Fetch data
         csets_and_members: Dict[str, List[Dict]] = fetch_cset_and_member_objects(
             codeset_ids=list(failed_cset_ids), flag_issues=False)
+        # TODO: mark discarded drafts resolved
```

Comment:
## TODO: mark discarded drafts resolved
Otherwise, `fetch_failures_0_members` will keep running w/ every refresh (every 20 min) because it will think that the discarded draft is still an issue.

I'm going to do this in a new PR.